components/PersonalSponsors.vue

```diff
@@ -76,5 +89,12 @@ export default {
       letter-spacing: 0.93px;
     }
   }
+  .PersonalSponsors-Names {
+    margin-right: 20px;
+    .PersonalSponsors-Sama {
+      margin-left: 3px;
+      font-size: 13px;
```

Comment:
```suggestion
      margin-left: 3px;
      font-size: 13px;
      font-weight: 500;
```
細かいんですが、太さ変えた方が読みやすいかと